.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: python
 python:
   - "2.7"
+cache: pip
+
 install:
-  - 'pip install websocket-client pytest'
+  - pip install flake8 pytest websocket-client
+
+before_script:
+  # stop the build if there are Python syntax errors or undefined names
+  - flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+  # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
+  - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
The github editor size seems like a pretty arbitrary limit. Maybe go for 120 instead, which is a bit more common? (I'd go for 100 or 80, but that gives a bit too many warnings at the moment, so let's decrease it further at a later time.)